snuba_sdk/mql/mql.py

```diff
@@ -13,10 +13,14 @@
 from snuba_sdk.column import Column
 from snuba_sdk.conditions import And, BooleanCondition, BooleanOp, Condition, Op, Or
 from snuba_sdk.formula import ArithmeticOperator, Formula
-from snuba_sdk.metrics_query import MetricsQuery
-from snuba_sdk.query_visitors import InvalidQueryError
 from snuba_sdk.timeseries import Metric, Timeseries
 
+
+# In order to avoid circular imports, we need to redefine the exception here.
```

Comment:
you could just define an `exceptions` module as well and then you don't have to redefine